src/varanus/__main__.py

```diff
@@ -9,6 +9,7 @@
 
 from .__version__ import __version__
 from .session import Session
+from .__patch__ import apply_patch
```

Comment:
Wouldn't this look more pleasing vertically if the patch import was above the session import?